vagrant/provision.sh

```diff
@@ -39,3 +47,7 @@ fi
 
 # final setups
 ./vagrant/provisioning_scripts/setup_js_env.sh
+
+# copy libreoffice service's conf
+sudo cp /home/vagrant/src/ralph/vagrant/upstart/soffice.conf /etc/init/soffice.conf
+sudo start soffice
```

Comment:
I extracted LibreOffice's stuff and move to separated script (``vagrant/provisioning_scripts/libre_office.sh``)